src/components/form-helpers/form-helpers.js

```diff
@@ -155,13 +155,16 @@ const ErrorMessage = injectIntl(({ intl, error }) =>
 );
 
 export const FieldItem = connect(
-  ({ className, fieldName, children, formik: { errors, touched } }) => (
+  ({ className, fieldName, children, formik: { errors, touched, submitCount } }) => (
     <li
-      className={concatClasses(className, touched[fieldName] && errors[fieldName] ? 'error' : '')}
+      className={concatClasses(
+        className,
+        submitCount && touched[fieldName] && errors[fieldName] ? 'error' : '',
```

Comment:
if you put in fieldItem its apply for all forms isnt??

Reply:
Almost. Because we are not using _FieldItem_ in all forms.

I searched for _touched_ and I also found _UpgradePlanForm_.

Reply:
but if in future want one form validate before first submit need change right? its for understand only

Reply:
Right, in that scenario we could avoid using _FieldItem_, or we could use a flag in _FieldItem_.